src/utils/api.ts

```diff
@@ -1,6 +1,7 @@
 import { stringify } from 'qs';
 
-export const apiUrl = 'https://api.quran.com/api/v3';
+export const ITEMS_PER_PAGE = 10;
+export const { NEXT_PUBLIC_API_HOST } = process.env;
```

Comment:
What is this variable meant to do? Is it the just the API URL?

Reply:
Yes, 'NEST_PUBLIC` prefix will expose the variable to browser as well.